core/jazz_ui/src/environments/environment.oss.ts

```diff
@@ -1,33 +1,33 @@
 export const environment = {
   production: true,
-  INSTALLER_VARS: {INSTALLER_VARS},
+  INSTALLER_VARS: { "environment_tabs": { "overview": true, "code_quality": "false", "deployments": true, "assets": true, "logs": true }, "CREATE_SERVICE": { "DEPLOYMENT_TARGETS": { "API": { "options": [{ "label": "AWS API Gateway", "value": "aws_apigateway" }, { "label": "GCP APIGEE", "value": "gcp_apigee" }], "active": true }, "FUNCTION": { "options": [{ "label": "AWS Lambda", "value": "aws_lambda" }], "active": false }, "WEBSITE": { "options": [{ "label": "AWS S3", "value": "aws_s3" }, { "label": "AWS Cloudfront", "value": "aws_cloudfront" }], "active": false } } }, "feature": { "multi_env": true, "apigee": "false" }, "service_tabs": { "overview": true, "cost": false, "access_control": true, "metrics": true, "logs": true } },
   configFile: 'config/config.oss.json',
-  baseurl: "https://{API_GATEWAY_KEY_PROD}.execute-api.{inst_region}.amazonaws.com/prod",
-  api_doc_name: "https://{api_doc_name}.s3.amazonaws.com",
+  baseurl: "https://dbn9ruqpn3.execute-api.us-east-1.amazonaws.com/prod",
+  api_doc_name: "https://multi0312-jazz-s3-api-doc-20190312053238719000000007.s3.amazonaws.com",
```

Comment:
remove hardcoded values, and in other places

Reply:
Removed